getVfRep.py

```diff
@@ -0,0 +1,49 @@
+import common
+from logger import logger
+from testConfig import TestConfig
+from thread import ReturnValueThread
+from task import Task
+from host import Result
+from typing import Optional
+import sys
+import yaml
+import json
+import jc
+
+class GetVfRep(Task):
```

Comment:
I think this class name needs a bit of work. We aren't trying to GetVfRep. Getting the VF rep is a requirement, but what we are really want this plugin to do is to get the VF rep stats and traces.

Reply:
Agreed with William's point